xendit.php

```diff
@@ -1223,69 +1223,37 @@ static function getNotificationUrl ($order) {
+		);
 	}
 
 	/**
      * Map Xendit failure message. Return failure title and message.
      * @param string $failure_reason
      * @return array
      */
-	static function getXenditFailureMessage ($failure_reason) {
+	static function getXenditFailureMessage ($failure_reason = '') {
         switch ($failure_reason) {
 			case 'CARD_DECLINED':
+				return 'Card declined by the issuer bank. Please try with another card or contact the bank directly. Code: 200011';
 			case 'STOLEN_CARD':
-				return array(
-					'title' => 'CARD_DECLINED',
-					'message' => 'The bank that issued this card declined the payment but didn\'t tell us why. Try another card, or try calling your bank to ask why the card was declined.'
-				);
+				return 'Card declined by the issuer bank. Please try with another card or contact the bank directly. Code: 200013';
 			case 'INSUFFICIENT_BALANCE':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'Your bank declined this payment due to insufficient balance. Ensure that sufficient balance is available, or try another card'
-				);
+				return 'Card declined due to insufficient balance. Ensure sufficient balance is available, or try another card. Code: 200012';
 			case 'INVALID_CVN':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'Your bank declined the payment due to incorrect card details entered. Try to enter your card details again, including expiration date and CVV'
-				);
+				return 'Card declined due to incorrect card details. Please try again. Code: 200015';
 			case 'INACTIVE_CARD':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'This card number does not seem to be enabled for eCommerce payments. Try another card that is enabled for eCommerce, or ask your bank to enable eCommerce payments for your card.'
-				);
+				return 'Card declined by the issuer bank. Please try with another card or contact the bank directly. Code: 200014';
 			case 'EXPIRED_CARD':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'Your bank declined the payment due to the card being expired. Please try another card that has not expired.'
-				);
+				return 'Card declined due to expiration. Please try again with another card. Code: 200010';
 			case 'PROCESSOR_ERROR':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'We encountered issue in processing your card. Please try again with another card.'
-				);
+				return 'We encountered an issue while processing the checkout. Please try again. Code: 200009';
 			case 'AUTHENTICATION_FAILED':
-				return array(
-					'title' => $failure_reason,
-					'message' => 'Authentication process failed. Please try again'
-				);
+				return 'The authentication process failed. Please try again. Code: 200001';
             default:
-				return array(
-					'title' => $failure_reason,
-					'message' => 'Failed to process transaction.'
-				);
+				return 'Failed to process transaction.';
```

Comment:
Wanna use this default error message instead? 
`We encountered an issue while processing the checkout. Please try again. Code: 100007' `